### PR TITLE
admin_users

### PR DESCRIPTION
**Pull request by Baranovsky1986**
Create a new user account, login, logout, and view all users via endpoints.

### CODE REVIEW

File: users/views.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from django.shortcuts import render
+
+from rest_framework import generics
+
+from . import models
+from . import serializers
+
+class UserListView(generics.ListAPIView):
```
Comment (criticus) [inline users/views.py:8]: is there a way to get a specific user?

Comment (Baranovsky1986) [reply users/views.py:8]: Do you mean to get user by id?

Comment (criticus) [reply users/views.py:8]: yes

Comment (Baranovsky1986) [reply users/views.py:8]: it can be added, but at first it would be nice to merge it

Comment (Baranovsky1986) [reply users/views.py:8]: in order to avoid conflicts at this stage, as a basic application that can then be improved

**Pull request merged**